### PR TITLE
initial commit of two services

### PR DESCRIPTION
**Pull request by ranum**
Did some code cleanup but did not merge or change what we had earlier.  Wanted to give you code as soon as possible.

kafkahelper is the kafka tool
fhirnotifier is the notification service

### CODE REVIEW

File: services/kafkahelper/kafkahelper.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    if 'topic' in request.args:
+        topic = request.args.get("topic")
+
+    if len(topic) == 0: #topic arg not present so do topiclist
+        consumer = KafkaConsumer(bootstrap_servers=kafkabootstrap,
+                                 sasl_mechanism="PLAIN", sasl_plain_username=kafkauser, sasl_plain_password=kafkapw)
+
+        existingtopics = consumer.topics()
+
+        message = {
```
Comment (ibmatc) [inline services/kafkahelper/kafkahelper.py:44]: Nit - could use a method to generate responses

generateResponse(statusCode=200, data=[]):
  message = {
    "status": status,
  }
  message.update(data)
  resp = jsonify(message)
  resp.status_code= status
  return resp

Then you could replace calls with something like:
  
generate_response(200, {"topics": str(existingtopics)})
or
generate_response(400, {"message": "Topic not found: selected topic does not exist"})

and the response packaging would stay consistent

File: services/kafkahelper/kubernetes.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
```
Comment (ibmatc) [inline services/kafkahelper/kubernetes.yml:1]: I would recommend NOT delivering this with your PR. It will be generalized and covered by my Helm chart.

**Review by ibmatc — APPROVED** [review]
LGTM - a couple of minor comments/suggestions.  Main concern is finalizing a name.

File: services/kafkahelper/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# kafkahelper
+
+A deployable service to assist with kafka interaction
+
+- listing topics
+
+(GET)  https://kafkahelperbaseurl
```
[the remark this answers is not in the document]
Comment (ranum) [reply services/kafkahelper/README.md:7]: I think \<foo> will allow that in markdown, I'll make the change

[the remark this answers is not in the document]
Comment (ranum) [reply services/kafkahelper/README.md:7]: sure

**Pull request closed without merging**